modules/claims_api/app/controllers/claims_api/concerns/request_logging.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module RequestLogging
+    extend ActiveSupport::Concern
+    # rubocop:disable Metrics/BlockLength
+    included do
+      before_action :log_request
+
+      def log_request
+        if @current_user.present?
+          hashed_ssn = Digest::SHA2.hexdigest @current_user.ssn
+          Rails.logger.info('Claims App Request', 'lookup_identifier' => hashed_ssn)
+        else
+          hashed_ssn = Digest::SHA2.hexdigest ssn
+          Rails.logger.info('Claims App Request',
+                            'consumer' => consumer,
+                            'va_user' => requesting_va_user,
+                            'lookup_identifier' => hashed_ssn)
+        end
+      end
+
+      def log_response(additional_fields = {})
+        logged_info = {
+          'consumer' => consumer,
+          'va_user' => requesting_va_user
+        }.merge(additional_fields)
+        Rails.logger.info('Claims App Response', logged_info)
+      end
+
+      def consumer
```

Comment:
Do you think this is the right class to have the header methods? If we need to use the LOA header, I don't think the Request logging concern would be the right place. Maybe we should have module specifically for header logic?

Reply:
mmm good idea, I'll look into that